scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="sno-du-enabling-workload-partitioning-on-single-node-openshift"]
+= Workload partitioning on single node OpenShift
+include::modules/common-attributes.adoc[]
+:context: sno-du-enabling-workload-partitioning-on-single-node-openshift
+
+toc::[]
+
+In resource-constrained environments, such as single node production deployments, it is advantageous to reserve most of the CPU resources for your own workloads and configure {product-title} to run on a fixed number of CPUs within the host. In these environments, management workloads, including the OpenShift control plane, need to be configured to use fewer resources than they might by default in normal clusters. You can isolate the {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+
+When you use workload partitioning, the CPU resources used by {product-title} for cluster management are isolated to a partitioned set of CPU resources on a single node cluster. This partitioning isolates cluster management functions to the defined number of CPUs. All cluster management functions operate solely on that `cpuset`.
+
+The minimum number of reserved CPUs required for the management partition for a single node cluster is four CPU Hyper threads (HTs). Inclusion of Operators or workloads outside of the set of accepted management pods requires additional CPU HTs. The set of pods that make up the baseline OpenShift installation and a set of typical add-on Operators are annotated for inclusion in the management workload partition. These pods operate normally within the minimum size `cpuset`. Any additional pods annotated for inclusion in the management workload partition require additional CPU resources to be added to that partition.
```

Comment:
Be careful about plain OpenShift references. The are almost never [acceptable](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#openshift).